secio/src/handshake/handshake_context.rs

```diff
@@ -81,7 +81,12 @@ impl HandshakeContext<()> {
 
     // Setup local proposition.
     pub fn with_local(self) -> HandshakeContext<Local> {
+        #[cfg(not(target_arch = "wasm32"))]
```

Comment:
There are many conditional compilation flags to select random methods. Creating a `rand` module using `getrandom` can encapsulate the differences in only one place.

Reply:
done